app/lib/screens/OnboardingScreens/Signup.dart

```diff
@@ -18,225 +22,220 @@ class SignupScreen extends StatefulWidget {
 
 class _SignupScreentate extends State<SignupScreen> {
   final _formKey = GlobalKey<FormState>();
-  final emailController = TextEditingController();
+  final userNameController = TextEditingController();
   final passwordController = TextEditingController();
+  final tokenController = TextEditingController();
```

Comment:
Usually tokens get generated and not required by user to input. Is this for testing purposes?

Reply:
I think the secret access token will be handed out to the user. Isn't that right? @gnunicorn